site/docs/tools/producer_tools.md

```diff
@@ -0,0 +1,7 @@
+# Producer Tools
+
+## Isthmus
+
+[Substrait Isthmus](https://github.com/substrait-io/substrait-java/tree/main/isthmus) is a Java 
+library which enables serializing SQL to [Substrait Protobuf](https://substrait.io/serialization/binary_serialization/) 
+via the Calcite SQL compiler.
```

Comment:
```suggestion
[Isthmus](https://github.com/substrait-io/substrait-java/tree/main/isthmus) is an application 
that serializes SQL to [Substrait Protobuf](https://substrait.io/serialization/binary_serialization/) 
via the Calcite SQL compiler.
```
I think the fact that it uses Java under the hood is irrelevant.  It tries (via graal) to produce a simple executable so the user would never have to know that they were using Java.  Now, it's not easily built and requires cloning substrait-java but I think those details are irrelevant in a tool description.